### ISSUE TITLE
 VIVO Harvester (xsltranslator issue) 

### ISSUE THREAD

**Issue by joseagaleano** [vivo-project/VIVO-Harvester#38]
I am using VIVO Harvester on CentOS7 with VIVO 1.9.3. I have copied the example-oaifetch and modified it according to my configuration. When I execute "run-oaifetch.sh" it runs without any errors and it reaches the last line but as I check the terminal I see this:

```
[root@localhost oaifetch]# harvester-xsltranslator -X xsltranslator.config.xml
2018-05-25 07:37:15.305 INFO  [o.v.h.t.XSLTranslator] XSLTranslator: Start
2018-05-25 07:37:15.308 DEBUG [o.v.h.u.a.ArgList] running XSLTranslator
2018-05-25 07:37:15.309 DEBUG [o.v.h.u.a.ArgList] command line args: -X xsltranslator.config.xml
2018-05-25 07:37:15.313 DEBUG [o.v.h.u.a.ArgList] config file args: --output translated-records.config.xml --input raw-records.config.xml --xslFile oaifetch-mets.datamap.xsl --force --wordiness DEBUG
2018-05-25 07:37:15.402 DEBUG [o.v.h.u.r.RecordHandler] 'rhClass' - 'org.vivoweb.harvester.util.repo.TextFileRecordHandler'
2018-05-25 07:37:15.402 DEBUG [o.v.h.u.r.RecordHandler] 'fileDir' - 'data/raw-records'
2018-05-25 07:37:15.403 DEBUG [o.v.h.u.r.RecordHandler] Using class: 'org.vivoweb.harvester.util.repo.TextFileRecordHandler'
2018-05-25 07:37:15.415 DEBUG [o.v.h.u.r.RecordHandler] 'rhClass' - 'org.vivoweb.harvester.util.repo.TextFileRecordHandler'
2018-05-25 07:37:15.415 DEBUG [o.v.h.u.r.RecordHandler] 'fileDir' - 'data/translated-records'
2018-05-25 07:37:15.415 DEBUG [o.v.h.u.r.RecordHandler] Using class: 'org.vivoweb.harvester.util.repo.TextFileRecordHandler'
2018-05-25 07:37:15.416 DEBUG [o.v.h.u.r.TextFileRecordHandler] Directory 'data/translated-records' Does Not Exist, attempting to create
2018-05-25 07:37:15.418 DEBUG [o.v.h.u.r.TextFileRecordHandler] Directory 'data/translated-records/.metadata' Does Not Exist, attempting to create
2018-05-25 07:37:15.438 DEBUG [o.v.h.u.r.TextFileRecordHandler] Compiling list of records
2018-05-25 07:37:15.441 DEBUG [o.v.h.u.r.TextFileRecordHandler] List compiled
2018-05-25 07:37:15.441 INFO  [o.v.h.t.XSLTranslator] 0 records translated.
2018-05-25 07:37:15.441 INFO  [o.v.h.t.XSLTranslator] 0 records did not need translation
2018-05-25 07:37:15.442 INFO  [o.v.h.t.XSLTranslator] XSLTranslator: End
```

When I check the raw-data folder I see many files without an extension. Anyone knows what could be wrong?

**Comment by TheRedDev** [comment]
raw-records or raw-data? If it's not that simple, you can email me (chris@chrishaines.net) and I'll try to help.

**Comment by joseagaleano** [comment]
The files are in raw-records

**Comment by fereira** [comment]
The files in raw-records are the result of the first step in the harvest pipeline from running the OAIFetch tool.  They don't have file extensions, but are basically xml files.  Each file corresponds to one article.  The harvester-xsltranslator tool will transform the xml (using the xsl file defined in xsltranslator.config.xml) to transform the oai-pmh result into vivo compliant RDF.    While all of the harvester tools can be run with command line options, the examples use {toolname}.config.xml files to define all the parameters.  The "Wordiness" parameter is basically a log level and you can try changing it to DEBUG to get more info.  The xsltranslator.config.xml file has a "force" parameter set to true (unless you've changed it).  Unless you clean out the data directory, it won't transform data in the raw-records file if there are transformed files in the data/translated-records directory.

**Comment by joseagaleano** [comment]
The data/translated-records folder is empty, before I run the script I delete the data folder and sub-folders. Even after I run the script the data/translated-records folder is empty.
In the terminal output I pasted on the original issue, translator is on debug.
The force parameter I didn't add or change, since I copied the example-oaifetch I only modified certain parts, but that parameter I haven't touched.
